EmbedCases.cmake

```diff
@@ -50,4 +50,9 @@ endmacro (opm_pack_stone)
 opm_pack_stone (upscale_relperm_benchmark)
 opm_pack_case (upscale_relperm_benchmark benchmark20)
 
-add_custom_target(benchmarks DEPENDS upscale_relperm_benchmark)
+if(INSTALL_BENCHMARKS)
+	add_custom_target(benchmarks ALL DEPENDS upscale_relperm_benchmark)
```

Comment:
alternative is that i have to lift the option up to the opm-common level, since it's opm-common that adds the target (ADDITIONAL_SOURCES) - and these are excluded from all.